client/src/ui/modules/entity-details/EntityDetails.tsx

```diff
@@ -1,7 +1,7 @@
 import { useMemo } from "react";
 import { useLocation } from "wouter";
-import { CombatEntityDetails } from "./CombatEntityDetails";
 import { BuildingEntityDetails } from "./BuildingEntityDetails";
+import { CombatEntityDetails } from "./CombatEntityDetails";
```

Comment:
are we using different linter ? why are there some places where lines order are changed. ?